src/data/list/alist.lean

```diff
@@ -8,11 +8,21 @@ import data.list.sigma
 /-!
 # Association Lists
 
-This file makes use of the contents of the `data.list.sigma` file to define association lists. These
-association lists are represented by the `alist` structure, which comprises a list of sigmas as a
-key-value store, as well as a proof that this has no duplicate keys.
+This file defines [association lists](https://en.wikipedia.org/wiki/Association_list). An
+association list is a list where every element consists of a key and a value, and no two entries
```

Comment:
Mentioning `sigma` in the dosctring would be nice. Just saying that it's `{ l : list (sigma B) // l.nodup }` is the best way to get someone who knows about Lean but nothing about association lists to understand it.

Reply:
I guess so, but I adhere to the philosophy that if the best way to understand a piece of code is to read the code itself, then why copy the code into the documentation?